code/modules/research/rdconsole.dm

```diff
@@ -498,28 +501,37 @@ won't update every console in existence) but it's more of a hassle to do. Also,
 					message_admins("Circuit imprinter exploit attempted by [key_name(usr, usr.client)]!")
 
 				if(g2g) //Again, if input is wrong, do nothing
-					add_wait_message("Imprinting Circuit. Please Wait...", IMPRINTER_DELAY)
+					var/time_to_construct = IMPRINTER_DELAY //Flat 1.6 seconds

```

Comment:
Casting a var purely to use it as a define...which you never modify, seems pointless.